src/grg_sphinx_theme/theme/grg_sphinx_theme/static/styles/components/navbar.css

```diff
@@ -1,5 +1,5 @@
 /* Customizing the navbar dropdown */
-.bd-header .navbar-nav .dropdown .dropdown-menu {
+.bd-header ul.navbar-nav .dropdown .dropdown-menu {
```

Comment:
Can you have both to be backward compatible ? Just in case we need to go back to an older version and for the older build

Reply:
This will work on backwards